bodhi/__init__.py

```diff
@@ -61,8 +61,8 @@ def get_user(request):
 
 
 def groupfinder(userid, request):
-    user = request.user
-    if user:
+    if request.user:
+        user = User.get(request.user.name, request.db)
```

Comment:
User with capital U seems to be only imported in get_user(...), maybe missing an import here or need that on a global level?